Some documentation updates (#2367)
nspcc-dev/neofs-node: README.md

make images
```

# Running

## CLI

`neofs-cli` allows to perform a lot of actions like container/object management
connecting to any node of the target network. It has an extensive description
for all of its commands and options internally, but some specific concepts
have additional documents describing them:
 * [Sessions](docs/cli-sessions.md)
 * [Extended headers](docs/cli-xheaders.md)

`neofs-adm` is a network setup and management utility usually used by the

> network administators. Refer to [docs/cli-adm.md](docs/cli-adm.md) for mode

Reviewer: ```suggestion
network administrators. Refer to [docs/cli-adm.md](docs/cli-adm.md) for mode
```

Author: Fixed.

environment variables.

See [docs/sighup.md](docs/sighup.md) on how nodes can be reconfigured without
restart.

See [docs/storage-node-configuration.md](docs/storage-node-configuration.md)
on how to configure a storage node.

### Example configurations

These examples contain all possible configurations of NeoFS nodes. All
parameters are correct there, however, their particular values are provided
for informational purposes only (and not recommended for direct use), real
networks and real configuration are likely to differ a lot for them.

 See [node.yaml](node.yaml) for configuration notes.
- Storage node
  - YAML (with comments): [node.yaml](config/example/node.yaml)
  - JSON: [node.json](config/example/node.json)
  - Environment: [node.env](config/example/node.env)
- Inner ring
  - YAML: [ir.yaml](config/example/ir.yaml)
  - Environment: [ir.env](config/example/ir.env)

# Private network

If you're planning on NeoFS development take a look at
[neofs-dev-env](https://github.com/nspcc-dev/neofs-dev-env/). To develop
applications using NeoFS we recommend more light-weight
[neofs-aio](https://github.com/nspcc-dev/neofs-aio) container. If you really

> want to get your hands dirty refer to [docs/deploy.md](docs/deploy.md) for

Reviewer 2: ```suggestion
want to do it all by hand refer to [docs/deploy.md](docs/deploy.md) for
```
sounds negative to me
